Cinema system/Cinema system/Controllers/OrdersController.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using CinemaSystem.Interfaces;
+using CinemaSystem.Models;
+
+namespace CinemaSystem.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class OrdersController : ControllerBase
+    {
+
+        private readonly IOrderService _orderService;
+
+        private const string ErrorOfUserNonexistence = "This user does not exists.";
+        private const string ErrorOfOrderNonexistence = "This order does not exists.";
+        private const string ErrorOfSeanceNonexistence = "This seance does not exists.";
+        private const string ErrorOfSeatBooked = "This seat does not exists or has already booked";
+
+        public OrdersController(IOrderService orderService)
+        {
+            _orderService = orderService;
+        }
+
+        [HttpGet("{userId}")]
+        public IActionResult GetOrders(int userId)
+        {
+            if (!_orderService.CheckUserExists(userId))
+            {
+                return NotFound(new Response(ErrorOfUserNonexistence));
+            }
+
+            var orders = _orderService.GetOrders(userId);
+
+            return Ok(new GetResponse<IEnumerable<OrderView>>(orders));
+        }
+
+        [HttpGet("{userId}/user/{orderId}")]
```

Comment:
It's better to use users/{userId}/orders/{orderId} scheme for all actions in this controller as logically user can have many orders but order will always have only one user.